commands/kubernetes.go

```diff
@@ -897,6 +933,104 @@ func (s *KubernetesCommandService) RunKubernetesClusterDelete(c *CmdConfig) erro
 	return nil
 }
 
+func (s *KubernetesCommandService) RunKubernetesClusterDeleteSelective(c *CmdConfig) error {
+	err := ensureOneArg(c)
+	if err != nil {
+		return err
+	}
+	clusterIDorName := c.Args[0]
+
+	clusterID, err := clusterIDize(c.Kubernetes(), clusterIDorName)
+	if err != nil {
+		return err
+	}
+
+	update, err := c.Doit.GetBool(c.NS, doctl.ArgClusterUpdateKubeconfig)
+	if err != nil {
+		return err
+	}
+
+	force, err := c.Doit.GetBool(c.NS, doctl.ArgForce)
+	if err != nil {
+		return err
+	}
+
+
+	volumes, err := c.Doit.GetStringSlice(c.NS, doctl.ArgVolumeList)
+	if err != nil {
+		return err
+	}
+
+	volSnapshots, err := c.Doit.GetStringSlice(c.NS, doctl.ArgVolumeSnapshotList)
+	if err != nil {
+		return err
+	}
+
+	loadBalancers, err := c.Doit.GetStringSlice(c.NS, doctl.ArgLoadBalancerList)
+	if err != nil {
+		return err
+	}
+
+	if force || AskForConfirmDelete("Kubernetes cluster", 1) == nil {
```

Comment:
Just FYI: If we wanted to build out a message confirming the additional resources, `AskForConfirm` provides more flexibility.

Reply:
👍 Thank you! I think it's sufficient to say k8s cluster here though.